bin/build-css.sh

```diff
@@ -5,6 +5,7 @@
 
 # Build the application stylesheet
 ./node_modules/.bin/sass $@ \
+     --load-path=. \
```

Comment:
New version of mon-frontend refers to node_modules directory which is in the root of the project